Flow.Launcher.Core/Plugin/PluginManager.cs

```diff
@@ -32,27 +28,18 @@ public static class PluginManager
         // todo happlebao, this should not be public, the indicator function should be embeded 
         public static PluginsSettings Settings;
         private static List<PluginMetadata> _metadatas;
-        private static readonly string[] Directories = { Constant.PreinstalledDirectory, DataLocation.PluginsDirectory };
 
-        private static void ValidateUserDirectory()
```

Comment:
this is not needed? we would want to if exits right

Reply:
Do you mean the removed `Directory.Exists` check? If so, the [documentation of `Directory.CreateDirectory`](https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.createdirectory?view=netcore-3.1) implies that this check is redundant...